docs/javascript-interop.md

```diff
@@ -23,9 +23,68 @@ A Blazor app can invoke JavaScript functions from .NET and .NET methods from Jav
 
 There are times when Blazor .NET code is required to call a JavaScript function. For example, a JavaScript call can expose browser capabilities or functionality from a JavaScript library to the Blazor app.
 
-To call into JavaScript from .NET, use the `IJSRuntime` abstraction, which is accessible from `JSRuntime.Current`. The `InvokeAsync<T>` method on `IJSRuntime` takes an identifier for the JavaScript function you wish to invoke along with any number of JSON-serializable arguments. The function identifier is relative to the global scope (`window`). If you wish to call `window.someScope.someFunction`, the identifier is `someScope.someFunction`. There's no need to register the function before it's called. The return type `T` must also be JSON serializable.
+To call into JavaScript from .NET, use the `IJSRuntime` abstraction. The `InvokeAsync<T>` method on `IJSRuntime` takes an identifier for the JavaScript function you wish to invoke along with any number of JSON-serializable arguments. The function identifier is relative to the global scope (`window`). If you wish to call `window.someScope.someFunction`, the identifier is `someScope.someFunction`. There's no need to register the function before it's called. The return type `T` must also be JSON serializable.
 
-In the sample app, two JavaScript functions are available to the client-side app that interact with the DOM to receive user input and display a welcome message:
+For server-side Blazor apps:
+
+* Multiple user requests are processed by the server-side app. Don't call `JSRuntime.Current` in a component to invoke JavaScript functions.
+* Inject the `IJSRuntime` abstraction and use the injected object to issue JavaScript interop calls.
+
+The following example demonstrates how to invoke a JavaScript function that assigns HTML markup to an element in a server-side Blazor app component.
+
+Inside the `<head>` element of *wwwroot/index.html*, compose a function that:
+
+* Receives a reference to the element and the markup content.
+* Applies the content to the element by assigning to the element's `innerHTML` property.
+
+```html
+<script>
+  window.Index = {
+    div: {
+      init: function (elem, content) {
+        elem.innerHTML = content;
```

Comment:
I'd prefer to not show using JS interop to manipulate the DOM. In Blazor apps you typically shouldn't be messing with the DOM directly. Can we show calling some sort of browser API API instead?